### PR TITLE
Controllers spec

### PR DESCRIPTION
**Pull request by osbentech**
In this branch I created a request test for my controllers, testing:

-  If the response status was correct.
-  If a correct template was rendered.
-  If the response body includes correct placeholder text.

### CODE REVIEW

File: spec/requests/user_spec.rb [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+  it 'is not valid without a photo' do
+    subject.photo = nil
+    expect(subject).to_not be_valid
+  end
+
+  it 'is not valid without a bio' do
+    subject.bio = nil
+    expect(subject).to_not be_valid
+  end
```
Comment (tugsem) [inline spec/requests/user_spec.rb]: - [ ] Please test the show and index actions using the right matchers as requested in the project.
>describe 'GET /index' do
  ...
end

>describe 'GET /show' do
  ...
end
image

File: spec/requests/post_spec.rb [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+
+  it 'is not valid without a likes counter' do
+    subject.likes_counter = nil
+    expect(subject).to_not be_valid
+  end
+
+  it 'is not valid without a comments counter' do
+    subject.comments_counter = nil
+    expect(subject).to_not be_valid
+  end
```
Comment (tugsem) [inline spec/requests/post_spec.rb]: - [ ] Please test the show and index actions using the right matchers as requested in the project.
>describe 'GET /index' do
  ...
end

>describe 'GET /show' do
  ...
end
image

**Review by tugsem — CHANGES_REQUESTED** [review]
##  STATUS: REQUIRED CHANGES ♻️
Hello @osbentech 👋 , 

Good job so far! 👏 
There are some issues that you still need to work on to go to the next project but you are almost there!

Alt Text
## Highlights 🚀 
✔️linters are passing
✔️correct use of gitflow
✔️professional documentation
## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by mikethreels — APPROVED** [review]
Hi Benjamin, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- linters are passing
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**